pkg/generate-sabakan-machines/cmd/root.go

```diff
@@ -0,0 +1,139 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:          "generate-sabakan-machines CSV",
+	SilenceUsage: true,
+	Short:        "generate machines.json",
+	Long: `Generate machines.json from the CSV file for use with the 'sabactl create' command.
+Example:
+  generate-sabakan-machines CSV --machine-type-boot=r6525-boot-1 --machine-type-cs=r6525-cs-1 --machine-type-ss=r7525-ss-1
```

Comment:
`CSV` does not look like a csv file.

```suggestion
  generate-sabakan-machines input.csv --machine-type-boot=r6525-boot-1 --machine-type-cs=r6525-cs-1 --machine-type-ss=r7525-ss-1
```